src/routes/activityReports/handlers.js

```diff
@@ -71,7 +73,8 @@ async function sendActivityReportCSV(reports, res) {
           header: 'Creator',
         },
         {
-          key: 'collaborators',
+          // Alerts export should match what we show.
+          key: isAlerts ? 'alertCollaborators' : 'collaborators',
```

Comment:
I'm confused by this, why are they different?

Reply:
One is the export from alerts where we currently don't show the roles if they are missing. The other one will add the missing roles when exported from the approved AR table.